test/tasks/Regression.t.sol

```diff
@@ -331,6 +332,29 @@ contract RegressionTest is Test {
         _assertDataToSignNestedMultisig(multisigTask, actions, expectedDataToSign);
     }
 
+    /// @notice Expected call data and data to sign generated by manually running the DisableModule template at block 8098229 on sepolia
+    /// Simulate from task directory (test/tasks/example/sep/011-disable-module) with:
+    /// SIMULATE_WITHOUT_LEDGER=1 just --dotenv-path $(pwd)/.env --justfile ../../../../../src/improvements/single.just simulate
+    function testRegressionCallDataMatches_DisableModule() public {
+        string memory taskConfigFilePath = "test/tasks/example/sep/011-disable-module/config.toml";
+        // Call data generated by manually running the DisableModule template at block 8098229 on sepolia.
```

Comment:
I doubt the block on this numbers is correct as you don't have any block set on the .env of the task. It doesn't really matter as it's just a comment though